Makefile

```diff
@@ -15,13 +13,11 @@ requirements:
 production-requirements:
 	pip3 install -r requirements.txt
 
-test.install_elasticsearch:
```

Comment:
Can you update README to reflect the changes in this Makefile?

Reply:
Done.